xml/decoders/Fleischmann_Function.xml

```diff
@@ -14,38 +14,75 @@
 <!-- for more details.                                                      -->
 <decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder.xsd">
   <version author="Peter Mansvelder pman@mac.com" version="0.1" lastUpdated="20100825"/>
-  <!--
-     This decoder configuration file is based on ESU_LokPilotDCC.xml
-     V 4, dated 20050619, by Walter Thompson. Which also acknowledged
-     "the contribution of many others" including 'jake'.
-
-     Based on ....
-
-     This decoder XML is meant to be used with the "Comprehensive" programmer format.
-     Continued the practice of using unrelated "item" names to place ESU unique
-     variables on the proper pane of the Comprehensive programmer.
-
-     Any and all feedback on this file would be appreciated.
-
-     Version History:
-
-        0.1 - Initial release 25 August 2010
-
--->
+	<version author="Wouter Kok" version="1.00" lastUpdated="20190131"/>
+  <!-- Version 0.10 - Initial release 25 August 2010 (is unusable) -->
+	<!-- Version 1.00 - Initial version total rewritten              -->
   <decoder>
-    <family name="Fleischmann function decoders" mfg="Fleischmann" lowVersionID="10" comment="Fleischmann 686401">
-      <model model="686401" maxInputVolts="25 V" maxTotalCurrent="0.80 A">
+    <family name="Fleischmann function decoders" mfg="Fleischmann">
+      <model model="686401" lowVersionID="10" maxInputVolts="20V" maxTotalCurrent="0.80 A" formFactor="" connector="Wires" numOuts="4" numFns="9" productID="686401">
+			<output name="1" label="Output A">
+					<label xml:lang="de">Ausgang A</label>
+					<label xml:lang="nl">Uitgang A</label>
+				</output>
+				<output name="2" label="Output B">
+					<label xml:lang="de">Ausgang B</label>
+					<label xml:lang="nl">Uitgang B</label>
+				</output>
+				<output name="3" label="Output C">
+					<label xml:lang="de">Ausgang C</label>
+					<label xml:lang="nl">Uitgang C</label>
+				</output>
+				<output name="4" label="Output D">
+					<label xml:lang="de">Ausgang D</label>
+					<label xml:lang="nl">Uitgang D</label>
+				</output>
+				<size length="28" width="12.2" height="2.6" units="mm"/>
+      </model>
+			<model model="686403" lowVersionID="11" maxInputVolts="20V" maxTotalCurrent="0.80 A" formFactor="" connector="Wires" numOuts="4" numFns="4" productID="686403">
```

Comment:
Thanks. Can you check these indents so ``</model>`` closes aligned?
Also for the ``<output>`` tags. 1 "jump" seems enough to see the corresponding level.

Reply:
@silverailscolo Is this OK to merge?

Reply:
This ``<output>`` line (as well as line 23) deserve 1 indent. Both are inside the ``<model>`` tag but currently have 0 indent. With a jump in lines 23 and 42 both will align with their ``</output>`` closing tag.
The rest of the file is consistently aligned now. Thanks.

Reply:
Thanks @Re465